docs/uml/main/InputDatamodelConcept.puml

```diff
@@ -36,11 +36,20 @@ abstract Class ConnectorInput {
 }
 ConnectorInput --|> AssetInput
 
-abstract Class AssetCharacteristicsInput {
-+ type: String
+abstract Class AssetCharacteristicsInput<T extends AssetTypeInput> {
```

Comment:
Does not match implementation which is `SystemParticipantTypeInput`

Reply:
Adapted the implementation instead. ;-)